projects/hslayers/src/components/layermanager/layermanager-wmst.service.ts

```diff
@@ -110,39 +109,39 @@ export class HsLayerManagerWmstService {
    * @returns True for WMS layer with time support
    */
   layerIsWmsT(layer: HsLayerDescriptor | Layer): boolean {
-    if (layer?.layer) {
-      // case of HsLayerDescriptor, Layer would have 'sublayers' property instead
-      layer = layer.layer;
-    }
-    if (!layer) {
+    const olLayer: Layer = layer?.layer ? layer.layer : layer;
```

Comment:
Could have been

```suggestion
    const olLayer: Layer = layer.layer ?? layer;
```

But I am fine with your change as well 🙂

Reply:
Nice trick!
https://youtu.be/UmmOJx_Hxto?t=126